app/js/lib/foxbox.js

```diff
@@ -327,6 +326,25 @@ export default class Foxbox extends Service {
     });
   }
 
+  performServiceCommand(id, command, method = 'GET', state = undefined) {
+    return fetchJSON(`${this.origin}/services/${id}/${command}`, method, state);
+  }
+
+  getSecureURL(sourceURL) {
```

Comment:
Naming is hard.... But we need something like this to add ```auth``` token to the resource strings (like I mentioned before jpeg link pointing to the box, that should return JPEG directly).